extension/chrome/settings/index.ts

```diff
@@ -50,6 +50,10 @@ View.run(class SettingsView extends View {
     const uncheckedUrlParams = Url.parse(['acctEmail', 'page', 'pageUrlParams', 'advanced', 'addNewAcct']);
     this.acctEmail = Assert.urlParamRequire.optionalString(uncheckedUrlParams, 'acctEmail');
     this.page = Assert.urlParamRequire.optionalString(uncheckedUrlParams, 'page');
+    if (this.page && !/^(\/chrome|modules)/.test!(this.page as string)) {
+      void Ui.modal.error('<div data-test="page-param-modal">An unexpected value was found for the page parameter</div>');
```

Comment:
I'm not sure how the void works or what it does, but the no-floating-promises error wanted you to handle possible promise rejection with a `.catch(Catch.reportErr)` - the `Ui.modal.error(...)` returns a promise that may reject silently if not handled.

Reply:
The void operator turns anything null, so we would explicitly ignore the error if the promise was rejected. You're correct that's a bad idea, so instead now I'm logging the error to the console.